docs/src/start/device-monitoring.md

```diff
@@ -12,63 +12,48 @@ Using these metrics, you can monitor the health of devices
 and can proactively initiate actions in case the device seems to malfunction.
 Additionally, the metrics can be used to help the customer troubleshoot when problems with the device are reported.
 
-%%te%% uses the open source component [`collectd`](https://collectd.org/) to collect the metrics from the device.
-%%te%% translates the collected metrics from their native format to the [%%te%% JSON](../understand/thin-edge-json.md) format
+%%te%% uses the open source component [collectd](https://collectd.org/) to collect the metrics from the device.
+%%te%% translates the collectd metrics from their native format to the [%%te%% JSON](../understand/thin-edge-json.md) format
 and then into the [cloud-vendor specific format](../understand/tedge-mapper.md).
 
-Enabling monitoring on your device is a 3-steps process:
-1. [Install `collectd`](#install-collectd),
-2. [Configure `collectd`](#configure-collectd),
-3. [Enable %%te%% monitoring](#enable-thin-edge-monitoring).
+![device monitoring with collectd](images/collectd-metrics.png)
 
-## Install collectd
+## Install
 
-Device monitoring is not enabled by default when you install %%te%%.
-You will have to install and configure [`collectd`](https://collectd.org/) first.
+Device monitoring is not enabled by default, however it can be enabled using a community package, [tedge-collectd-setup](https://cloudsmith.io/~thinedge/repos/community/packages/?q=name%3A%27%5Etedge-collectd-setup%24%27), which will install [collectd](https://collectd.org/) and configure some sensible defaults including monitoring of cpu, memory and disk metrics.
```

Comment:
Wouldn't it be better to also mention that this community plugin is **not a must-have** but just eases/replaces the installation and configuration of collectd, which can even be done yourself by following the previous mentioned steps? So that an exitsting user of collectd knows what to do.

Also I hope this plugin won't overwrite the user's existing config, if he's got one.

Reply:
The user's that we've been seeing are less interested in the details, and just want fewer steps (and especially be able to deploy it using thin-edge.io itself).

The tedge-collectd-setup uses the package principles of "config" files, so it is up to the default apt settings to whether configuration is overridden or not (https://github.com/thin-edge/thin-edge.io/pull/2518)

Reply:
The following section details how to customize the collectd section anyway, and the `tedge-collectd-setup` package is not destructive to existing collectd installations, it will only enable and start the tedge-mapper-collectd service.